Made v2

PersonThing4982 opened the pull request:
![image](https://github.com/user-attachments/assets/05c917cf-054e-4184-8ed8-233a40d236de)

PersonThing4982 committed:
Delete preview directory

PersonThing4982 committed:
Delete .vscode directory

PersonThing4982 committed:
Delete modules directory

PersonThing4982 committed:
Delete Build Release.bat

PersonThing4982 committed:
Delete requirements.txt

PersonThing4982 committed:
Delete main.py

PersonThing4982 committed:
Delete icon.ico

PersonThing4982 committed:
Delete README.md

PersonThing4982 committed:
Made v2 with gui

PersonThing4982 committed:
Add files via upload

PersonThing4982 committed:
Delete __pycache__ directory

PersonThing4982 committed:
Delete downloaders/__pycache__ directory

PersonThing4982 committed:
Delete gui/__pycache__ directory

PersonThing4982 committed:
Delete utils/__pycache__ directory

PersonThing4982 committed:
Update .gitignore

Official-Husko commented:
Well.. holy shit this is impressive. I would like to know how many times while you were coding screamed at my code and my logic for things. I will look through the code and if all is good definitely approve it. Although i do think a terminal based version is still something that should be kept in some way. Some users might prefer that over a GUI variant.

Is this cross-platform compatible?

jaykobdetar committed:
Puts things how they should be

This is a much more  reasonable push, especially since I only really tested it with a couple sites, and none of the api ones, I will make sure to bring the gui up to functionality anywhere it is broken when I have time,  it was a very rushed job.

PersonThing4982 commented:
I may just be tired, but I don't really know what you mean by cross-platform  compatible, however, I will add a cli to the gui version at some point, I am not sure if it would be easier to adapt the current one or make a new one tbh, although there can always be 2 sets of downloaders I suppose in the worst case scenario.

Official-Husko commented:
> I may just be tired, but I don't really know what you mean by cross-platform compatible, however, I will add a cli to the gui version at some point, I am not sure if it would be easier to adapt the current one or make a new one tbh, although there can always be 2 sets of downloaders I suppose in the worst case scenario.

For now we can just use your new build for the gui one and my "legacy" one for the cli build. I was actually planning to rewrite all of this in golang in the near future since i am ditching python wherever i can.

About the Cross-Platform part, i meant if the current code works on both Linux and Windows or if its Windows only. The latest version from me has support for both.

Additionally, i think separating the cli and gui one would be the best. We can strip out the gui libraries that way and make the application more lightweight.

PersonThing4982 commented:
It  almost certainly wont work on linux, or at least I made no effort to that effect, I could definitely re-adapt my approach to just try and make a gui compatible with whatever downloaders and structures you already have since they work, and that would make making the gui as compatible much easier -  ultimately it is your call on what you think is best, I am curious, why are you ditching python?

Official-Husko commented:
> It almost certainly wont work on linux, or at least I made no effort to that effect, I could definitely re-adapt my approach to just try and make a gui compatible with whatever downloaders and structures you already have since they work, and that would make making the gui as compatible much easier - ultimately it is your call on what you think is best

If you didn't change much i think there is a good possibility _**It Just Works**_ on Linux. I am on Linux and can try it and probably make it fully Linux compatible if something is not working.

> I am curious, why are you ditching python?

It was my first programming language i learned back in like 2018 i think and i used it up to probably early to mid 2024. I have started to feel fatigued and somehow for my ideas it felt rather limited to an extend. Especially GUI wise. QT sucks in my opinion and other solutions such as Tkinter (which if i did see right is what you used here) or Tkinter-modern or what its called are if were honest outdated. I started to dislike coding with it and switched to golang and godot since i think for my future plans they are just better in everything. 

**TLDR**: I started to dislike coding with python and found Golang much more enjoyable and useful.

Official-Husko commented:
Something else that is important. I am in the works of changing all of my licenses on my projects to a new one similar to [this one](https://gist.github.com/Official-Husko/9059f65ca29b62f70bbf02f4869c4bb3) (Not 1:1 since that one is for the new shop framework im developing and have been working on for about a year now).

I still fully stand with and support open-source software but i do not like companies or other people taking source code to create something to sell it without making the source code public. Im still updating the licence a bit more to make it as bulletproof as possible.

This would also impact your V2 variant if that is okay with you. If you don't want that we can discuss other licenses or changes to my new one for this project specifically.

Official-Husko commented:
One question i do have though. How did you stumble across this project and why did you decide to invest the time and energy into making a v2 variant? I do not mean that as an insult im just curious because this is the first time somebody forked something i made and did so much work to make pretty much a new major release.

Is it because you genuinely use the software or found it useful even tho there are competitors like gallery-dl?

PersonThing4982 commented:
I honestly never heard of gallery-dl, I  was just trying to find a way to easily download full comics from multporn and found  your project, but wanted  a gui and to be able to que downloads, I generally tend to try and make a gui for most cli's I come across because I prefer it, and didn't really feel like continuing to look for alternatives since this worked well enough. 
I will agree python has very limited gui options, in projects where adding the dependency is justified I will tend to try and use PyWebui so I can just use css and html, but I completely understand why you would just move off python.

Official-Husko commented:
god fucking dammit i wrote "guy" instead of "gui" for the branch..

Official-Husko commented:
I will add github actions to compile it for linux and windows in the near future. Once that all works and it has no issues on linux i willl add it to the masters markdown so we can have both a cli and gui variants.
